stable/redmine/values.yaml

```diff
@@ -98,9 +98,28 @@ postgresql:
     size: 8Gi
 
 ## Kubernetes configuration
-## For minikube, set this to NodePort, elsewhere use LoadBalancer
+##   minikube: NodePort
+##   ingress: ClusterIP
+##   elsewhere: LoadBalancer
 ##
 serviceType: LoadBalancer
+## Control hosts connecting to "LoadBalancer" only
+serviceLoadBalancerSourceRanges:
+  - 0.0.0.0/0
+
+ingress:
+  # hostName: redmine.cluser.local
```

Comment:
if you make this value `hostname` it will be more consistent with the Ingress for Drupal and Wordpress: https://github.com/kubernetes/charts/blob/master/stable/drupal/values.yaml#L68